package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "ffc-ce-payment-orchestrator",
   "description": "Digital service mock to claim public money in the event property subsides into mine shaft.",
-  "version": "0.0.1",
+  "version": "1.1.0",
```

Comment:
As the previous version was `0.0.1` should the version be bumped to `1.0.0` or `0.1.0` instead of `1.1.0`?